wordpress-mobile/AztecEditor-Android
# Issue/366 remove excluded features

### Fix
Remove ***Preformat*** and ***Page Break*** from heading menu and format toolbar, respectively, as described in https://github.com/wordpress-mobile/AztecEditor-Android/issues/366.

### Test
1. Run demonstration app.
2. Tap ***Heading*** format button.
3. Notice ***Preformat*** is not in popup menu.
4. Tap away from popup menu.
5. Scroll format toolbar to the left.
6. Notice ***Page Break*** is not in format toolbar.
